backend/api/Models/Concepts/ResearchFile/ResearchFilePropertyMap.cs

```diff
@@ -0,0 +1,30 @@
+using Mapster;
+using Entity = Pims.Dal.Entities;
+
+namespace Pims.Api.Models.Concepts
+{
+    public class ResearchFilePropertyMap : IRegister
+    {
+        public void Register(TypeAdapterConfig config)
+        {
+            config.NewConfig<Entity.PimsPropertyResearchFile, ResearchFilePropertyModel>()
+                .PreserveReference(true)
```

Comment:
Is this needed to prevent a circular reference from research -> propertyresearch -> research?

Reply:
Yes